### DISCUSSION TITLE
Name & label of histograms

### DISCUSSION THREAD

**Discussion by dylanjaide** [scikit-hep/hist#377] [Q&A] [answered]
I'm having some trouble getting the name & label parameters for histograms to work when creating a new one with `hist.Hist()`. Here's a MWE with `hist` v2.5.2:

```
In [1]: import hist

In [2]: x = hist.axis.Regular(10, 0, 1, name="my_axis")

In [3]: h = hist.Hist(x, label="hist name", name="hist name")

In [4]: h
Out[4]: Hist(Regular(10, 0, 1, name='my_axis', label='my_axis'), storage=Double())
```

The name "hist name" is not present in the final hist at all. Is this intended? Is there some other way I should be giving histograms a name? Thanks :)

**Comment by amangoel185** [comment, marked as answer]
The histogram name is present, although not displayed. You can use `h.name` to access the name of the histogram. :)

Although, the name/label could be added to the repr too. Feel free to open an issue for the same!
